composer.json

```diff
@@ -4,49 +4,58 @@
     "type": "project",
     "description": "Symfony.se",
     "autoload": {
-        "psr-0": { "": "src/" }
+        "psr-4": {
+            "": "src/"
+        },
+        "classmap": [
+            "app/AppKernel.php",
+            "app/AppCache.php"
+        ]
     },
+
     "require": {
-        "php": ">=5.3.3",
-        "symfony/symfony": "2.3.*",
-        "doctrine/orm": ">=2.2.3,<2.4-dev",
-        "doctrine/doctrine-bundle": "1.2.*",
-        "twig/extensions": "1.0.*",
-        "symfony/assetic-bundle": "2.3.*",
-        "symfony/swiftmailer-bundle": "2.3.*",
-        "symfony/monolog-bundle": "2.3.*",
-        "sensio/distribution-bundle": "2.3.*",
-        "sensio/framework-extra-bundle": "2.3.*",
-        "incenteev/composer-parameter-handler": "~2.0",
-        "knplabs/knp-markdown-bundle": "1.2.*",
-        "dms/meetup-api-bundle": "~1.0",
-        "happyr/slugify-bundle": "1.0.*",
-        "debril/rss-atom-bundle": "~1.3",
-        "happyr/excerpt-bundle": "0.1.*"
+        "php": ">=5.3.9",
+        "symfony/symfony": "2.8.*",
+        "doctrine/orm": "^2.4.8",
+        "doctrine/doctrine-bundle": "^1.4",
+        "twig/twig": "^1.0||^2.0",
+        "twig/extensions": "^1.4.0",
+        "symfony/assetic-bundle": "^2.3",
+        "symfony/swiftmailer-bundle": "^2.3,>=2.3.10",
+        "symfony/monolog-bundle": "^3.0.2",
+        "sensio/distribution-bundle": "^4.0",
+        "sensio/framework-extra-bundle": "^3.0.2",
+        "incenteev/composer-parameter-handler": "^2.0",
+        "knplabs/knp-markdown-bundle": "^1.5.1",
+        "dms/meetup-api-bundle": "^1.0.0",
+        "happyr/slugify-bundle": "^1.0.1",
+        "debril/rss-atom-bundle": "^3.0.5",
```

Comment:
What were the BC breaks between 1.3.2 and ^3.0.5? I can see some that has not been taken care of.